cogs/commands/quotes.py

```diff
@@ -313,67 +307,111 @@ async def quote(self, ctx: Context, arg: QueryConverter = None):
         # send message with no pings
         await ctx.send(message, allowed_mentions=AllowedMentions().none())
 
-    @quote.command(help='Add a quote, format !quote add <author> "<quote text>".')
-    async def add(self, ctx: Context, author: MentionConverter, quote=None):
+    @quote.command()
+    async def add(self, ctx: Context, author: MentionConverter, *, quote):
+        """Add a quote, format !quote add <author> <quote text>"""
         requester = get_name_string(ctx)
         now = datetime.now()
 
-        result = add_quote(requester, author, quote, now)
+        try:
+            result = f"Thank you {requester}, recorded quote with ID #{add_quote(requester, author, quote, now)}."
```

Comment:
I think having the "work" be part of the string interpolation is a bit obscure. Having:
```py
quote_id = add_quote(requester, author, quote, now)
result = f"Thank you {requester.mention}, recorded quote with ID #{quote_id}."
```
Would make the code easier to reason about.